img/d/dab-bank.de.svg

```diff
@@ -0,0 +1 @@
00a770"/></linearGradient><linearGradient x1="56.64" y1="120.94" x2="56.64" y2="132.8" spreadMethod="pad" id="E" xlink:href="#G"><stop offset="0" stop-color="#00a770"/><stop offset=".3" stop-color="#80d2a4"/><stop offset=".55" stop-color="#00a770"/><stop offset=".84" stop-color="#007b50"/><stop offset=".88" stop-color="#007b50"/><stop offset="1" stop-color="#1ab07b"/></linearGradient><linearGradient x1="56.85" y1="123.18" x2="56.85" y2="133.73" spreadMethod="pad" id="F" xlink:href="#G"><stop offset="0" stop-color="#007b50"/><stop offset=".6" stop-color="#80d2a4"/><stop offset="1" stop-color="#00a770"/></linearGradient><linearGradient id="G" gradientUnits="userSpaceOnUse"/></defs><g transform="matrix(1.3229400000000002 0 0 1.3229400000000002 -61.4 -163.28)"><path d="M46.4 133.42h10v-10h-10z" fill="url(#F)"/><path d="M47.19 123.6c-.2 0-.32.16-.35.4s-.25 1.3-.25 4.1c0 2.78.25 4.6.25 4.6.02.23.15.4.35.4h8.65c.2 0 .35-.16.35-.4l-.1-4.22.1-4.49c0-.21-.16-.4-.35-.4z" fill="url(#E)"/><path d="M52.05 124.6c.27.12 1.37.7 1.5.9-.55.07-1.63.35-2.22.64l-.03.07c.64-.14 1.54-.22 2.16-.25-.23.34-.38.5-.74.88l-.03.08 1.48-.8c.23.35.29 1.14.3 1.78.2-.4.44-1.91.34-2.18.41-.29.47-.51.47-.51-.38.03-.58.04-.79.1a7.18 7.18 0 0 0-2.41-.78z" fill="url(#D)"/><path d="M47.95 129.29c-.05.5-.08.86 0 1.26-.2.06-.39.07-.55.13l-.03.07a7 7 0 0 0 .77.43c.16.33.32.56.58.82l.04-.07c-.05-.19-.11-.35-.1-.56.34.1.62.15 1 .16l.02-.07c-.32-.13-.65-.23-.92-.48.08-.23.18-.41.34-.64l.03-.07c-.28.04-.54.14-.78.19-.2-.34-.27-.8-.37-1.24z" fill="url(#C)"/><path d="M49.62 127s-.2-.03-.56-.02c-.3.02-.73.04-.95.1l-.03.08c.34.01.81.1.99.28s-.52.47-1.03 1.49.93-.87 1.33-1.09l.36 1.05.03-.07c.04-.22.14-.83.31-1.2s.06-.05.36-.07c.3-.03.72-.05 1.52.1l.03-.08a5.5 5.5 0 0 0-1.68-.52h.03c.1-.31.23-.54.23-.54-.16.02-.62.24-.94.48" fill="url(#B)"/><path d="m51.57 131.3-.03-.4-.47.59c-.2.05-.4.05-.6.04h-.23l-.03.08c.19.09.36.15.56.19-.16.14-.37.25-.51.34l-.03.07c.3-.07.72-.2 1.03-.39.15.01.3 0 .41-.02l.04-.07c-.02-.07-.09-.11-.1-.21.28-.23.49-.43.73-.67l.03-.07c-.23.16-.52.39-.8.52" fill="url(#A)"/></g><path d="M10.32 3.48 8.36 4.53c.52-.56.7-.8 1.02-1.27-.82.05-2.01.14-2.86.33.74-.4 2.2-.83 2.98-.94a10.24 10.24 0 0 0-2-1.18c1.16.09 2.56.52 3.24.93.27-.08.55-.08 1.05-.13-.13.22-.14.35-.62.67.13.36-.2 2.36-.46 2.88-.01-.83-.08-1.88-.4-2.34m-5.97 7.15c-.47-.17-.81-.37-1.18-.72.1-.31.24-.55.44-.86l-.99.16a5.08 5.08 0 0 1-.52-1.54c-.08.66-.1 1.14 0 1.67-.31.08-.52.16-.77.27l1.01.56c.21.44.43.75.78 1.09-.08-.3-.1-.53-.07-.84.43.13.8.2 1.3.21M5.56 4c-.15.19-.32.54-.36.7 1.2.17 2.18.8 2.18.8-1.06-.2-1.62-.17-2.01-.14s-.48.09-.48.09c-.28.62-.45 1.7-.45 1.7-.2-.47-.47-1.4-.47-1.4C3.43 6.04 2.2 7.2 2.2 7.2a7.01 7.01 0 0 1 1.36-1.96c-.24-.24-.86-.36-1.3-.38.23-.1.83-.2 1.29-.23l.74.01c.42-.32 1.04-.6 1.26-.63m1.28 5.8-.63.78c-.4.11-.73.15-1.13.15.24.13.48.2.73.26-.25.23-.47.39-.71.55.41-.1.95-.28 1.36-.52.21.01.4 0 .55-.03a3.17 3.17 0 0 1-.09-.37c.38-.3.66-.57.98-.89-.34.24-.63.41-1.02.6l-.04-.54" fill="#fff"/></svg>
```

Comment:
How about this one? It's from their [wikipedia](https://de.wikipedia.org/wiki/BNP_Paribas) page.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" style="background:#00925c;fill:#fff"><path d="M18.16 3.54c.88.37 4.4 2.29 4.84 2.89-1.87.25-5.43 1.3-7.23 2.25a48 48 0 0 1 6.92-.8 21.26 21.26 0 0 1-2.44 3.08C22.4 9.67 23 9.59 24.97 8.43c.74 1.1.93 3.65.96 5.67.65-1.28 1.43-6.11 1.1-6.99 1.2-.78 1.2-1.09 1.5-1.62-1.2.1-1.87.13-2.53.33-1.65-1.02-5.06-2.06-7.84-2.28Zm-4.69 6.13c-.53.08-2.05.76-3.05 1.55 0 0-.67-.08-1.78-.05-1.12.05-2.59.33-3.15.56 1.09.05 2.57.33 3.15.9 0 0-1.67 1.5-3.29 4.76 0 0 2.99-2.78 4.27-3.47 0 0 .66 2.26 1.13 3.38 0 0 .42-2.61 1.1-4.11 0 0 .2-.12 1.15-.21s2.29-.15 4.86.3c0 0-2.36-1.5-5.28-1.9.11-.4.52-1.26.89-1.71zm-8.4 8.87a13.4 13.4 0 0 0-.02 4.06c-.74.2-1.26.37-1.86.63.85.52 1.54.93 2.49 1.36.5 1.09 1 1.8 1.85 2.63-.2-.7-.24-1.27-.16-2.02 1.06.31 1.94.48 3.14.52a7.85 7.85 0 0 1-2.86-1.76c.27-.74.58-1.33 1.08-2.06-.92.13-1.6.24-2.4.4a12.4 12.4 0 0 1-1.26-3.76zm14.03 5c-.8.57-1.5 1-2.44 1.45-.05-.43-.07-.8-.1-1.29-.52.7-.96 1.24-1.54 1.88-.99.27-1.77.37-2.72.38.58.3 1.17.48 1.78.6-.61.56-1.14.96-1.74 1.34 1-.24 2.31-.67 3.29-1.24.5.02 1.01 0 1.36-.07-.08-.3-.15-.53-.23-.92.92-.71 1.58-1.36 2.34-2.13z"/></svg>
```

Reply:
For me, the green background only shows up in a browser, but not with image viewers/editors.

Reply:
Good thing we're publishing on a website then :stuck_out_tongue: